### ISSUE TITLE
Tile - Ensure color is not the sole means of communicating information - (2036570307)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5608]
### Module: 
 33 Tile 

### Violation: 
 Ensure color is not the sole means of communicating information 

![image](https://user-images.githubusercontent.com/5023024/202729133-1029e592-9121-41b1-81eb-5bfb6cf89bc8.png)

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2036570307 

### Severity: 
 10 

### Description: 
 [Issue]
There is a link after "Test 2. Tile with Link" heading that uses image with blue color to indicate that it is link.

[User Impact]
Users with color blindness will have difficulty distinguishing which text is a link.

[Code Reference] 
 ```
<a class="" href="https://developers.arcgis.com/calcite-design-system/components/tile">
  <div class="container">
    (...)
    <div class="content-container">
      <div class="content">
        <div class="heading">Rivers</div>
        <div class="description">A river is a natural flowing watercourse, usually freshwater, flowing towards an ocean, sea, lake or another river.</div>
      </div>
    </div>
  </div>
</a>
 ``` 

### Note: 
 [Suggestion]
Ensure color is not the sole means of communicating information. Color can be used as long as other visual indicators of the information are also present. Common additional visual indicators for links include underlining or bolding.
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
@SkyeSeitz + @ashetland - WDYT re: the `tile` component when it includes a link, in particular around the use of color? 

Is there more we could add design-wise to make the component pop a bit more when a link is present? 

When using JAWS I'm experiencing the same as the above screenshot, whereas on `:hover` the component contains an outline. Maybe we need to add the outline in to other states?

JAWS screen reader:
![image](https://user-images.githubusercontent.com/5023024/202730751-70016807-e291-41a4-bf69-b081e2d96130.png)


`:hover` state:
![image](https://user-images.githubusercontent.com/5023024/202730578-ced64031-86a8-44a1-8ca0-29cf83902692.png)

**Comment by geospatialem** [comment]
Dig it! Hoping for inclusion prior to the 1.0 release. 

@ashetland Would we be good to go to assign to devs with the above proposal?

**Comment by geospatialem** [comment]
@driskull @macandcheese Link context is provided, this is strictly a visual update when adding a `href` prop to provide additional visual context to users.

**Comment by driskull** [comment]
I will defer to the design team here. This is a design decision

**Comment by ashetland** [comment]
@SkyeSeitz do you have thoughts on the above? Trying to reach a consensus as this is a good v1 candidate.

**Comment by macandcheese** [comment]
The above recording doesn't seem to have the cursor change, or hover / press effects that the tile currently has when `href` is present... If we want to change the appearance of the idle state, we could explore that - would be in addition to the current hover / press states?

https://user-images.githubusercontent.com/4733155/203658837-21bd2c47-5cdf-4e03-9326-f3878b6b2c49.mov

**Comment by ashetland** [comment]
> The above recording doesn't seem to have the cursor change, or hover / press effects that the tile currently has when `href` is present... If we want to change the appearance of the idle state, we could explore that - would be in addition to the current hover / press states?
> 
>  Screen.Recording.2022-11-23.at.2.46.53.PM.mov

That gif is just a rough mock in Figma.

**Comment by SkyeSeitz** [comment]
Yes, I would imagine It would be in addition to the current hover / press states. Similar to the behavior below (please ignore that the icon changes color in this particular recording and that the target area is not the entire tile).
https://user-images.githubusercontent.com/29716057/203659375-537a7d97-43d5-4c36-98b2-eab6faf7459f.mov

**Comment by macandcheese** [comment]
There are two behaviors there 😄  - the left or the right? I guess I feel the underline is fine if it shows on hover, but my vote would be to not have it present at idle.

**Comment by ashetland** [comment]
But then we'd be back at the same problem of color being the only differentiator, right? Is there another Calcite convention that would work better?

**Comment by driskull** [comment]
The card that is linked looks too similar to the one without a link. The only difference is a color which users won't be able to differentiate.

IMO there needs to be something more than just a color change to tell users that one can be interacted with. Whether that's a border, underline, icon, or some combination. I'll defer to the design team there.

**Comment by SkyeSeitz** [comment]
> There are two behaviors there 😄 - the left or the right? I guess I feel the underline is fine if it shows on hover, but my vote would be to not have it present at idle.

This was to demonstrate the difference between the tiles on the right with links and the tiles on the left without links (thought they are still clickable 🙃), but I can see @ashetland and @driskull point that we may want a visual indicator before hover.

**Comment by macandcheese** [comment]
I guess it's weird to me we are trying to differentiate between a non-interactive tile and an interactive tile. 

What is the point of a tile that you cannot interact with (unless disabled...)? 

If people are using Tiles for non-clickable or focusable containers, that's really a use case for a card, right?

Really, IMO, the card should be styled the same regardless of whether or not there is a `href` present, it's just as likely an app is using the Tile with a click or keyboard event to do something other than navigate with href - like open a Modal.

**Comment by ashetland** [comment]
This is an excellent point. Reading our docs, though, it implies they can be used in a non-clickable state. Losing the underline animation also makes sense to me. 

<img width="851" alt="CleanShot 2022-11-23 at 15 21 20@2x" src="https://user-images.githubusercontent.com/108549080/203662534-e850ec6d-8a3e-47d6-8448-b349f1747de0.png">

**Comment by SkyeSeitz** [comment]
Three options that show a distinction between tiles with links and without. Our preference is the first option on the left as it leverages a recognizable underline convention, while not becoming visual cluttered.

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/29716057/203664139-3eaac64f-ca45-432d-954c-289c4743bbfb.png">

**Comment by macandcheese** [comment]
> Reading our docs, though, it implies they can be used in a non-clickable state.
> 

Yeah, to me this is incorrect. I think we could re-evaluate how we want these to be used - to me a non-interactive Tile isn't a use case for the component.

I think the first column above is a good update with our current functionality.

**Comment by macandcheese** [comment]
Not assigned yet - Along with the above updates, I do still think we should evaluate the need for a non-clickable Tile at all (except in a disabled state). That's a use case for a Card IMO.

**Comment by geospatialem** [comment]
To summarize the above discussion it sounds like in the short-term we're on board with adding styling to the `tile` where a link is present, correct? 

Regarding the use use cases @macandcheese mentioned, perhaps a future team discussion?

![image](https://user-images.githubusercontent.com/5023024/209863326-056c154a-e3c4-4242-8720-e8f01913ee00.png)

**Comment by ashetland** [comment]
Correct

**Comment by geospatialem** [comment]
Seeking final design input pertaining to `tile`, as it relates to our [Enhancement requests](https://confluencewikidev.esri.com/display/Calcite/Enhancement+Requests) *(internal link)* to address at a later time for:

>  Audit of Tile component / Selection Modes - Update to Tile Group and Tile

**Comment by geospatialem** [comment]
Verified on `1.1.1-next.2`

![image](https://user-images.githubusercontent.com/5023024/227383536-f8f60298-02dd-4535-ad7f-865fe6791316.png)
